packetbeat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Beats dockerfile used for testing
+FROM ruflin/beat-tester
```

Comment:
Strange, as the build is from yesterday :-( Is it somehow cached or did I configure it wrong? https://hub.docker.com/r/ruflin/beat-testing/tags/

About the line: Agree, the goal is to get rid of it and use a common image to inherit from.